infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/Containers.java

```diff
@@ -116,4 +116,96 @@ public static void addRamRequest(Container container, String limitInK8sNotion) {
     container.setResources(
         resourceBuilder.addToRequests("memory", new Quantity(limitInK8sNotion)).build());
   }
+
+  /**
+   * Returns the CPU limit in cores, if it is present in given container otherwise 0 will be
+   * returned.
+   */
+  public static float getCpuLimit(Container container) {
```

Comment:
This is an simle getter-like method, so not very handy to make it optional, since it will 1) require boxing 2) prevents from direct usage in other method calls (since Optional method arguments are not good)